CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,44 @@
+## Linkerd Community Code of Conduct
+
+As a CNCF project, we follow the [CNCF Contributor Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+Additionally, we are committed to the following guidelines adapted
+from the [Rust Code of Conduct](https://www.rust-lang.org/en-US/conduct.html):
+
+### Community Guidelines
+
+Our goal is to foster an inclusive and diverse community of technology
+enthusiasts.
+
+Try to be your best self. Treat your fellow community members with kindness
+and empathy. We welcome disagreements when they are conducted respectfully and
+without personal attacks.
+
+We ask that you keep unstructured critique to a minimum. Disparaging remarks
+about the project are unnecessary and a drain on community morale. Feedback
+should be constructive and relevant. Having passionately held opinions on what
+should improve is encouraged! We hope you will use that enthusiasm to roll up
+your sleeves and get involved by submitting pull requests. We have additional
+guidelines on
+[how to ask constructive questions](https://github.com/linkerd/linkerd/wiki/How-To-Ask-Questions-in-Slack).
+
+We don't tolerate insults, spamming, trolling, flaming, baiting, or
+harassment. We don't tolerate sexual language, imagery, or unwanted advances.
+Private harassment is also unacceptable.
+
+We do our best to avoid [subtle-isms](https://www.recurse.com/manual#sub-sec-social-rules):
+small actions that make others feel uncomfortable. If you
+witness a subtle-ism, you may respectfully point it out to the person publicly
+or privately, or you may ask a moderator to say something. Accidentally saying
+something biased is common, expected, and readily forgiven. It is not in and
+of itself a bannable offense.
+
+### Moderation
+
+If you feel that a channel needs moderation, please message one of the
+following moderators directly: @alex, @siggy, @olix0r, @kl.
```

Comment:
Is this list up-to-date? E.g., kl is currently on internal project. Also wondering if we should use something like cncf-linkerd-dev@lists.cncf.io?